linkedlist/detect_loop.py

```diff
@@ -0,0 +1,50 @@
+def detect_loop(head):
```

Comment:
I believe, we can do this without extra space, using 'fast' and 'slow' pointers.